modules/creating-kubefed-CRs.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * kubefed/installing-kubefed.adoc
+
+[id="creating-kubefed-CRs_{context}"]
+= Creating KubeFed custom resources
+
+You must create the `KubeFedWebHook` and `KubeFed` custom resources (CRs) for {KubeFedProductShortName} deployment.
+The {KubeFedProductName} operator will create the required custom resource definitions (CRDs) for these CRs automatically.
+
+.Prerequisites
+* You will need to have the `oc` CLI tool installed.
+
+.Procedure
+
+. Create a `KubeFedWebHook` resource to instantiate an admission webhook controller for KubeFed. You must create this CR with `scope: Cluster` to allow the webhook controller to serve API requests coming from all KubeFed control planes within the same cluster.
++
+----
+$ cat <<-EOF | oc apply -n <namespace> -f -
+ ---
+ apiVersion: operator.kubefed.io/v1alpha1
+ kind: KubeFedWebHook
+ metadata:
+   name: kubefed-webhook-resource
+ spec:
+   scope: Cluster
+---
+----
+
+. Create a `KubeFed` resource to drive the installation of KubeFed. If you  planning to federate a cluster-scoped resource type, for example `StorageClass`, create this CR with `scope: Cluster`.
```

Comment:
s/If you  planning/If you planning/
more than one space character.

Reply:
Should be "are planning", so it's a typo, good catch!